pages/event.php

```diff
@@ -165,6 +165,47 @@ function event() {
         }
     }
 
+    if (isset($_GET['exchange'])) {
+        try {
+            // Change the item ID values based on dev/prod
+            // These are placeholder exchange options
+            switch ($_GET['exchange']) {
+                case "red_yen":
+                    $player->getInventory();
+                    if (!$player->hasItem("19")) {
```

Comment:
Could we code the Item ID in event config somewhere? That will make it easier to swap for dev/prod